README.md

```diff
@@ -1,12 +1,10 @@
 # react-a11y-announcer
 
-Welcome to React Announcer from Think Company.
-
-This is a React component that demonstrates how to dynamically add announcements to a web page, and ensure that they are accessible and announced properly by screen readers.
+`react-a11y-announcer` is a React a11y component that helps ensure announcements are accessible and properly announced by screen readers. 
```

Comment:
Consider linking the first instance of `a11y` to a definition of the term -- possibly [wikipedia](https://en.wikipedia.org/wiki/Computer_accessibility)